src/Roave/SecurityAdvisories/VersionConstraint.php

```diff
@@ -164,8 +164,8 @@ private function containsLowerBound(?Boundary $otherLowerBoundary) : bool
             return false;
         }
 
-        if (($this->lowerBoundary->limitIncluded() === $otherLowerBoundary->limitIncluded())
-            || $this->lowerBoundary->limitIncluded()
+        if ($this->lowerBoundary->limitIncluded()
```

Comment:
I think the first expression should be put into a variable

Reply:
fixed